src/app/Providers/components/VMwareProviderHostsTable/VMwareProviderHostsTable.tsx

```diff
@@ -57,6 +58,18 @@ const VMwareProviderHostsTable: React.FunctionComponent<IVMwareProviderHostsTabl
     }
   );
 
+  const commonNetworkAdapters = (hosts: IHost[]) =>
+    hosts[0].networkAdapters.filter(({ ipAddress }) =>
+      hosts.every((host) => host.networkAdapters.some((na) => na.ipAddress === ipAddress))
+    );
+
+  const hasCommonNetworkAdapters = () => {
+    if (selectedItems.length > 0) {
+      return commonNetworkAdapters(selectedItems).length > 0 ? true : false;
+    }
+    return false;
+  };
+
```

Comment:
Sorry to nitpick after my prior approval @gildub , but I realized this could be simpler. Remember that each time the selected items change, the whole component re-renders and these functions are reinitialized. They are always called with the same values and will return the same thing in all calls of a given render. So they don't need to be functions. It could just be one variable:

```ts
const commonNetworkAdapters =
  selectedItems.length > 0
    ? selectedItems[0].networkAdapters.filter(({ ipAddress }) =>
        selectedItems.every((host) => host.networkAdapters.some((na) => na.ipAddress === ipAddress))
      )
    : [];
```

And then below instead of `!hasCommonNetworkAdapters()` you could use `commonNetworkAdapters.length === 0`, and instead of passing `commonNetworkAdapters(selectedItems)` you can just pass `commonNetworkAdapters`.

It's simpler, and it has the benefit of only running the filter function once per render (currently it'll run 3 times).

Reply:
That's pretty slick!